core/metrics-core-service/src/main/java/org/hawkular/metrics/core/service/MetricsServiceImpl.java

```diff
@@ -940,11 +940,7 @@ public void setDefaultTTL(int defaultTTL) {
 
     @Override
     public Observable<Boolean> idExists(final MetricId<?> metricId) {
```

Comment:
+1 to getting rid of it. It was part of the API from the early part of the project and I believe used with the old metrics explorer. I am not aware of any other uses.